spec/rails_helper.rb

```diff
@@ -13,6 +13,15 @@
 require 'view_component/test_helpers'
 require 'view_component/system_test_helpers'
 
+Capybara.register_driver :selenium_chrome_headless do |app|
+  options = Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument('--headless')
+  options.add_preference(:download, default_directory: 'tmp/test_downloads')
```

Comment:
not sure it's worth it, but the directory name could be something in settings, or a constant avail to any test

Reply:
made it a constant, good call